packages/components/src/components/text-field-wrapper/text-field-wrapper.examples.md

```diff
@@ -250,16 +253,19 @@ export default class Code extends Vue {
     return this.typePassword.replace(/label="Some label"/, '$& show-password-toggle="false"');
   }
 
-  typeSearch =
-`<p-text-field-wrapper label="Some label">
+  submitButton = true;
+  submitButtons = [true, false];
+  get typeSearchMarkup() {
+    return `<p-text-field-wrapper label="Some label" submit-button="${this.submitButton}">
```

Comment:
This example is a bit confusing: if `submit-button="true"` it does not show a submit button. Though this is intended, it is somehow difficult to understand. Maybe we can enhance the description above to make it more clearer, that the text-field becomes never a submit button if it is used outside a `form` element. Or we delete the prop on the first example?